Source/Diagnostics/MultiDiagnostics.cpp

```diff
@@ -24,11 +24,7 @@ MultiDiagnostics::MultiDiagnostics ()
         if ( diags_types[i] == DiagTypes::Full ){
             alldiags[i] = std::make_unique<FullDiagnostics>(i, diags_names[i]);
         } else if ( diags_types[i] == DiagTypes::BackTransformed ){
-#ifdef WARPX_DIM_RZ
```

Comment:
Let's print a warning about the fields in RZ, while we have not implemented them? :)